diff -burNp duplicity-0.6.08/src/backends/sshbackend.py duplicity-0.6.08-dud/src/backends/sshbackend.py
--- duplicity-0.6.08/src/backends/sshbackend.py	2010-03-07 21:56:35.000000000 +0100
+++ duplicity-0.6.08-dud/src/backends/sshbackend.py	2010-03-11 16:51:58.525987063 +0100
@@ -31,7 +31,7 @@ import time
 import duplicity.backend
 from duplicity import globals
 from duplicity import log
-from duplicity import pexpect
+import pexpect
 from duplicity.errors import * #@UnusedWildImport
 
 class SSHBackend(duplicity.backend.Backend):
