--- duplicity-0.6.19/duplicity/backends/_ssh_pexpect.py.orig	2012-06-22 22:04:38.879374844 +0200
+++ duplicity-0.6.19/duplicity/backends/_ssh_pexpect.py	2012-06-22 22:04:47.079330420 +0200
@@ -32,7 +32,7 @@ import os
 import duplicity.backend
 from duplicity import globals
 from duplicity import log
-from duplicity import pexpect
+import pexpect
 from duplicity.errors import * #@UnusedWildImport
 
 class SSHPExpectBackend(duplicity.backend.Backend):
